code-blast.js

```diff
@@ -225,33 +225,36 @@ https://twitter.com/JoelBesada/status/670343885655293952
 			canvas.style.position = 'fixed';
 			canvas.style.top = 0;
 			canvas.style.left = 0;
-			canvas.style.zIndex = 1;
+			canvas.style.zIndex = 2147483647;
```

Comment:
I can see why this is needed, but maybe you should change this to a more reasonable number? :wink: 
For example, 42 or 9001 (over 9000!) :smile: